docs/source/ubuntu-opens-install.rst

```diff
@@ -53,17 +53,16 @@ Configuring the APT software sources
 
 The machines need to be configured such that APT can get access to the
 Calico packages. This step differs depending on whether you're using
-Icehouse or Juno.
-
-Icehouse
-^^^^^^^^
+Icehouse, Juno or Kilo.
 
 Add the Calico PPA.
 
 ::
 
-    sudo apt-add-repository ppa:project-calico/icehouse
+    sudo apt-add-repository ppa:project-calico/<release>
+
 
+Where <release> is icehouse, juno or kilo
```

Comment:
Note: to get \<release> to show up in a monospaced font, add backticks, like so: <code>\`\`\<release>``</code>. Also, this sentence needs a full stop.